src/app/mod.rs

```diff
@@ -71,7 +74,25 @@ impl<C: Client + Send + Sync + 'static> App<C> {
     }
 
     pub fn subscription(&self) -> Subscription<Message> {
-        self.state.subscription()
+        Subscription::batch(vec![
+            iced_native::subscription::events().map(Message::Event),
+            self.state.subscription(),
+        ])
+    }
+
+    pub fn should_exit(&self) -> bool {
+        self.should_exit
+    }
+
+    pub fn stop(&mut self) -> Command<Message> {
+        self.should_exit = true;
+        if self.context.internal_daemon {
+            return Command::perform(
+                state::cmd::stop(self.context.revaultd.clone()),
```

Comment:
We shouldn't kill the daemon when we close the GUI, as the daemon might be working in background for e.g. cpfp-ing transactions.

(Let's also make sure that the GUI can start again using the old daemon after a stop)

cc @darosior

Reply:
The gui stop the daemon only if it started it internally. About the cpfp-ing transactions, we have the same problem if a user is switching off it's computer. If the gui does not stop its internal daemon gracefully, the daemon will be stop by the process kill of the user.

Reply:
>  What do you suggest otherwise, leave the GUI open until the spend is confirmed?

I mean, we should suggest that. And then don't kill the daemon in case managers are really fucking stupid

Reply:
because the daemon is a in a std thread,  Stopping the gui (main thread) is killing the daemon (child thread) too, i introduce the `stop` only to stop it gracefully.